templates/tools/dockerfile/test/python_stretch_35_x64/Dockerfile.template

```diff
@@ -0,0 +1,29 @@
+%YAML 1.2
+--- |
+  # Copyright 2018 gRPC authors.
+  #
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+  
+  FROM debian:stretch
+  
+  <%include file="../../apt_get_basic.include"/>
+  <%include file="../../gcp_api_libraries.include"/>
+  
+  RUN apt-get update && apt-get install -y python2.7 python3.5 python-all-dev python3-all-dev
+
+  RUN curl https://bootstrap.pypa.io/get-pip.py | python2.7
+  RUN curl https://bootstrap.pypa.io/get-pip.py | python3.5
```

Comment:
Can we consolidate all the Python images into one (or two include files) with the only line different being `RUN curl https://bootstrap.pypa.io/get-pip.py | python3.5` where `python3.5` will be the Python version to be tested under?